File: SDKs/googleapis.com/speech/v1p1beta1/go/pkg/models/shared/createphrasesetrequest.go
package shared

// CreatePhraseSetRequest
// Message sent by the client for the `CreatePhraseSet` method.
type CreatePhraseSetRequest struct {
	PhraseSet   *PhraseSet `json:"phraseSet,omitempty"`
	PhraseSetID *string    `json:"phraseSetId,omitempty"`
}

File: SDKs/googleapis.com/speech/v1p1beta1/typescript/src/sdk/models/shared/createphrasesetrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { PhraseSet } from "./phraseset";



// CreatePhraseSetRequest
/** 
 * Message sent by the client for the `CreatePhraseSet` method.
**/
export class CreatePhraseSetRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=phraseSet" })
  phraseSet?: PhraseSet;

  @SpeakeasyMetadata({ data: "json, name=phraseSetId" })
  phraseSetId?: string;
}
